src/endpoints/user-status/data/dtos/userstatus.dto.ts

```diff
@@ -0,0 +1,18 @@
+export type GetUserStatustDto = {
+	walletId: string;
+};
+
+export enum DAOUserStatusType {
+	EXPLORER = 'Explorer',
+	MEMBER = 'Member',
+	CANDIDATE = 'Candidate',
+	CUSTODIAN = 'Custodian',
+}
+export type DaoUserStatuses = {
+	name: string;
+	status: DAOUserStatusType;
+};
+export enum ERROR_MESSAGE_TYPE {
```

Comment:
I understand the need here to have a custom error type. 
Probably, we should define it as a separate error type similar to what we have here

https://github.com/Alien-Worlds/eosdac-api/blob/master/src/common/api/domain/errors/limit-exceeded.error.ts